e2e/cypress/package-lock.json

```diff
@@ -45,7 +45,7 @@
         "localforage": "1.10.0",
         "lodash.intersection": "4.4.0",
         "lodash.mapkeys": "4.6.0",
-        "lodash.set": "^4.3.2",
+        "lodash.set": "4.3.2",
```

Comment:
Could you please revert this change?
Was it creating any issues for you?

Reply:
Done.
It was added automatically by `npm install`.